docs/apis-clients/web-modeler-api/index.md

```diff
@@ -0,0 +1,76 @@
+---
+id: index
+title: Web Modeler API (REST, beta)
+description: "Web Modeler API (beta) is a REST API and provides access to Web Modeler data.
+Requests and responses are in JSON notation."
+---
+
+:::caution Beta Offering
+Web Modeler API is currently offered as a [beta release](../../reference/early-access#beta). It is not recommended for production use and there is no maintenance service guaranteed.
+
+While in beta, the API may introduce breaking changes without prior notice.
+
+However, we encourage you to provide feedback via your designated support channel or the [Camunda Forum](https://forum.camunda.io/).
+:::
+
+Web Modeler provides a REST API at `/api/*`. Clients can access this API by passing a JWT access token in an authorization header `Authorization: Bearer <JWT>`.
+
+## OpenAPI documentation
+
+A detailed API description is available as [OpenAPI](https://www.openapis.org/) specification at [https://modeler.cloud.camunda.io/swagger-ui/index.html](https://modeler.cloud.camunda.io/swagger-ui/index.html).
+
+## Authentication
+
+To authenticate for the API, generate a JWT token and pass it in each request; guidance on this is provided in the following sections.
+
+### Obtain JWT token
+
+**Example:**
+
+1. Create client credentials by clicking **Console > Manage (Organization) > Console API > Create New Credentials**.
+2. Add permissions to this client for **Web Modeler API (beta)**.
```

Comment:
Are there specific steps to accomplish this?

Reply:
Yes, but trivial ones. Once you create credentials you will see a UI with check boxes for Web Modeler. 
![image](https://user-images.githubusercontent.com/14032870/209138743-143d6490-6afc-4bc2-a93c-06bae9c761ab.png)

Screenshot might be helpful, but it is pretty much self explanatory, in my mind. Also since this is the first iteration of documentation on a beta feature, I think it can be a little rough around the edges.

Personally, I would prefer a page like this https://docs.camunda.io/docs/components/console/manage-clusters/manage-api-clients/ for Console API clients.

The page I linked is about managing API clients on a per-cluster basis. There is also a way to manage clients/credentials on an organization-wide basis. For this, there is currently no documentation. If there would be, I would link to it.